src/ProjectNeon/Assets/Features/Effects/AllEffects.cs

```diff
@@ -8,7 +8,7 @@ public static class AllEffects
     {
         { EffectType.Nothing, e => new NoEffect() },
         { EffectType.HealFlat, e => new SimpleEffect(m => m.GainHp(e.IntAmount))},
-        { EffectType.PhysicalDamage, e => new PhysicalDamage(e.IntAmount) },
+        { EffectType.PhysicalDamage, e => new DamageApplied(new PhysicalDamage(e.IntAmount)) },
```

Comment:
`e.IntAmount` is wrong for a multiplier. All multipliers are floats.

Reply:
@SilasReinagel My bad sorry. Fixed

Reply:
This still isn't fixed.